papaya-integration-tests/src/test/resources/pl/edu/agh/papaya/tests/projects/Projects.md

```diff
@@ -34,9 +34,12 @@ is [added to the project](- "addToProject(#project.id, 'peter')").
 Now, [ ](- "c:echo=switchUser('peter')") should
 [have access to the project](- "c:assert-true=200==tryGetProjectById(#project.id).statusCode").
 
-[It is not possible](- "c:assert-true=400==tryAddToProjectNonExistingUser(#project.id).statusCode")
+[It is not possible](- "c:assert-true=400==tryToAddUserToProject(#project.id, 'i.do.not.exist@example.com').statusCode")
 to add a non-existing user to the project.
 
+[It is not possible](- "c:assert-true=409==tryToAddUserToProject(#project.id, 'peter.griffin@hotmail.com').statusCode")
```

Comment:
A to są chyba te sławne testy integracyjne

Reply:
Tutaj przekazałbym tylko username (`peter`) a w metodzie użyłbym `getUser(username)` żeby nie powtarzać wszędzie maili. Czyli chyba jednak trzeba będzie rozdzielić na dwie metody.